spec/migrations/20180626125654_make_maintenance_zone_record_spec.rb

```diff
@@ -0,0 +1,80 @@
+require_migration
+
+describe MakeMaintenanceZoneRecord do
+  let(:zone_stub) { migration_stub(:Zone) }
+
+  let(:remote_region_start) do
+    anonymous_class_with_id_regions.rails_sequence_start +
+      anonymous_class_with_id_regions.rails_sequence_factor
+  end
+
+  migration_context :up do
+    before(:each) do
+      zone_stub.delete_all
```

Comment:
Not sure why you need this...the database *should* be empty.

Reply:
it is not cleaned between specs in `migration_context` when I use `zone_stub.create!`. I don't know if it is some bad local settings, but specs are failing without this line.